### ISSUE TITLE
Roles page - inline form to invite user

### ISSUE THREAD

**Issue by smirolo** [djaodjin/djaoapp#77]
Currently the form to invite a user that is not yet registered is to pop up a modal dialog box. It seems the invite would look best if the form showed inline between the "add input" and the list of roles.

**Comment by knivets** [comment]
Should this be applied to accessibles page too? Should I just transfer all the contents of the modal inside an inline container?

**Comment by knivets** [comment]
<img width="525" alt="Screen Shot 2019-10-24 at 22 43 14" src="https://user-images.githubusercontent.com/1903479/67519520-acc7dd80-f6af-11e9-8eb5-b88ab3c61d94.png">

**Comment by smirolo** [comment]
![67519520-acc7dd80-f6af-11e9-8eb5-b88ab3c61d94](https://user-images.githubusercontent.com/3732869/67527765-99475300-f66c-11e9-9099-fd37c54c44a1.png)
1. Make sure there is only one button named "Invite Profile Manager".
2. When the user is not yet registered, add the "error" message (red text) and invite text area between the text input where the e-mail address have been entered and the invite button.

**Comment by smirolo** [comment]
Yes. Inline the dialog in "Connected profiles" the same way. Just be careful that the type ahead will be an `Organization` in that case.
